packages/active-campaign-client/src/helpers/manageListSubscription.ts

```diff
@@ -0,0 +1,40 @@
+import { APIGatewayProxyResult } from 'aws-lambda';
+import { acClient } from '../clients/activeCampaignClient';
+
+export async function manageListSubscription(
+  cognitoUserId: string,
```

Comment:
```suggestion
  cognitoId: string,
```

Reply:
this could be no longer necessay if u first do the refactor soggested in later comments